src/fw-update/fw-update-device.cpp

```diff
@@ -112,6 +139,21 @@ namespace librealsense
         return false;
     }
 
+    float update_device::compute_progress(float progress, float start, float end, float threshold) const
+    {
+        if (threshold < 1.f && threshold > -1.f)
```

Comment:
if (std::fl_abs(threshold) >= 1.f)

Do we allow a negative threshold ? will this work?
Maybe we can just say >= 1?
Just saying :)

Reply:
Just a style comment, your call